lix/packages/sdk/src/change-queue.test.ts

```diff
@@ -9,7 +9,10 @@ test("should use queue and settled correctly", async () => {
 		key: "mock-plugin",
 		glob: "*",
 		diff: {
-			file: async ({ old }) => {
+			file: async ({ old, neu }) => {
+				// TODO fix the data file parsing
+				// const parsedOld = new TextDecoder().decode(old?.data);
+				// const parsedNeu = neu ? new TextDecoder().decode(neu?.data) : undefined;
```

Comment:
The change queue test did not account for returning the encoded data. I added todos to update the test. THIS IS NOT RELATED TO REMOVING COMMITS